### ISSUE TITLE
crash on loading manga details page and about page

### ISSUE THREAD

**Issue by goldbattle** [nekomangaorg/Neko#1857]
### Steps to reproduce

Click on any manga, or from the three dot menu click the "about".

### Expected behavior

Not crash.

### Actual behavior

Crashed.

### Crash logs

```
App version: 2.16.8 (standard, f928b2950, 8, 2024-06-27T19:00:09.353600783)
Android version: 14 (SDK 34)
Android build ID: UP1A.231005.007.G990USQSBGXF1
Device brand: samsung
Device manufacturer: samsung
Device name: r9q
Device model: SM-G990U
Device product name: r9qsqw
           ******************************************************************************************************************************************************************************************************************************
           Exception that caused crash
           ******************************************************************************************************************************************************************************************************************************
           java.lang.Throwable: java.lang.IllegalStateException: CompositionLocal LocalLifecycleOwner not present
at androidx.lifecycle.compose.LocalLifecycleOwnerKt$LocalLifecycleOwner$1$1.invoke(SourceFile:9)
at kotlin.SynchronizedLazyImpl.getValue(Unknown Source:20)
at androidx.compose.material3.ButtonKt.read(SourceFile:35)
at androidx.compose.runtime.ComposerImpl.consume(SourceFile:1)
at coil.util.-Logs.collectAsStateWithLifecycle(SourceFile:11)
at eu.kanade.tachiyomi.ui.manga.MangaDetailController.ScreenContent(SourceFile:36)
at eu.kanade.tachiyomi.ui.base.controller.BaseComposeController$onViewCreated$1$1.invoke(SourceFile:2)
at eu.kanade.tachiyomi.ui.base.controller.BaseComposeController$onViewCreated$1$1.invoke(SourceFile:1)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at ColumnHeaderKt$$ExternalSyntheticOutline0.m(SourceFile:73)
at coil.request.Svgs.CompositionLocalProvider(SourceFile:24)
at androidx.compose.material3.TextKt.ProvideTextStyle(SourceFile:76)
at androidx.compose.material3.BadgeKt$Badge$1$1.invoke(SourceFile:89)
at androidx.compose.material3.BadgeKt$Badge$1$1.invoke(SourceFile:12)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at ColumnHeaderKt$$ExternalSyntheticOutline0.m(SourceFile:73)
at coil.request.Svgs.CompositionLocalProvider(SourceFile:47)
at androidx.compose.material3.ButtonKt.MaterialTheme(SourceFile:315)
at org.nekomanga.presentation.theme.ThemeKt.NekoTheme(SourceFile:80)
at eu.kanade.tachiyomi.ui.base.controller.BaseComposeController$onViewCreated$1.invoke(SourceFile:2)
at eu.kanade.tachiyomi.ui.base.controller.BaseComposeController$onViewCreated$1.invoke(SourceFile:1)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at androidx.compose.ui.platform.ComposeView.Content(SourceFile:25)
at ToolTipComponentsKt$ToolTipButton$2$1.invoke(SourceFile:76)
at ToolTipComponentsKt$ToolTipButton$2$1.invoke(SourceFile:16)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at ColumnHeaderKt$$ExternalSyntheticOutline0.m(SourceFile:73)
at coil.request.Svgs.CompositionLocalProvider(SourceFile:47)
at androidx.compose.ui.platform.CompositionLocalsKt.ProvideCommonCompositionLocals(SourceFile:263)
at androidx.compose.material.ButtonKt$Button$3.invoke(SourceFile:26)
at androidx.compose.material.ButtonKt$Button$3.invoke(SourceFile:9)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at ColumnHeaderKt$$ExternalSyntheticOutline0.m(SourceFile:73)
at coil.request.Svgs.CompositionLocalProvider(SourceFile:47)
at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt.ProvideAndroidCompositionLocals(SourceFile:458)
at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(SourceFile:5)
at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(SourceFile:1)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at ColumnHeaderKt$$ExternalSyntheticOutline0.m(SourceFile:73)
at coil.request.Svgs.CompositionLocalProvider(SourceFile:24)
at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(SourceFile:14)
at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(SourceFile:2)
at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
at androidx.compose.material3.ButtonKt.invokeComposable(SourceFile:1)
at androidx.compose.runtime.ComposerImpl.doCompose(SourceFile:119)
at androidx.compose.runtime.ComposerImpl.composeContent$runtime_release(SourceFile:11)
at androidx.compose.runtime.CompositionImpl.composeContent(SourceFile:34)
at androidx.compose.runtime.Recomposer.composeInitial$runtime_release(SourceFile:35)
at androidx.compose.runtime.CompositionImpl.composeInitial(SourceFile:1)
at androidx.compose.runtime.CompositionImpl.setContent(Unknown Source:2)
at androidx.compose.runtime.Latch$await$2$2.invoke(SourceFile:142)
at androidx.compose.ui.platform.WrappedComposition.setContent(SourceFile:16)
at androidx.compose.ui.platform.WrappedComposition.onStateChanged(Unknown Source:18)
at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(SourceFile:32)
at androidx.lifecycle.LifecycleRegistry.addObserver(SourceFile:241)
at androidx.compose.runtime.Latch$await$2$2.invoke(SourceFile:138)
at androidx.compose.ui.platform.AndroidComposeView.onAttachedToWindow(SourceFile:103)
at android.view.View.dispatchAttachedToWindow(View.java:23227)
at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3698)
at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3705)
at android.view.ViewGroup.addViewInner(ViewGroup.java:5552)
at android.view.ViewGroup.addView(ViewGroup.java:5328)
at android.view.ViewGroup.addView(ViewGroup.java:5268)
at android.view.ViewGroup.addView(ViewGroup.java:5240)
at com.bluelinelabs.conductor.changehandler.AnimatorChangeHandler.performChange(Unknown Source:34)
at com.bluelinelabs.conductor.ControllerChangeHandler.executeChange(SourceFile:189)
at com.bluelinelabs.conductor.Router.performControllerChange(SourceFile:13)
at com.bluelinelabs.conductor.Router.performControllerChange(SourceFile:4)
at com.bluelinelabs.conductor.Router.pushController(Unknown Source:13)
at eu.kanade.tachiyomi.ui.library.LibraryController.onItemClick(SourceFile:69)
at eu.davidea.viewholders.FlexibleViewHolder.onClick(Unknown Source:51)
at android.view.View.performClick(View.java:8047)
at android.view.View.performClickInternal(View.java:8024)
at android.view.View.-$$Nest$mperformClickInternal(Unknown Source:0)
at android.view.View$PerformClick.run(View.java:31890)
at android.os.Handler.handleCallback(Handler.java:958)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:230)
at android.os.Looper.loop(Looper.java:319)
at android.app.ActivityThread.main(ActivityThread.java:8919)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:578)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)

```

### Neko version

2.16.8

### Android version

14

### Device

Galaxy

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/nekomangaorg/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by PanisSupraOmnia** [comment]
My crash logs as well:
![neko_crash_logs.txt](https://github.com/user-attachments/files/16021470/neko_crash_logs.txt)

**Comment by nonproto** [comment]
fixed in 2.16.9  you may need to manually update
